pkg/transformers/namespace.go

```diff
@@ -97,5 +97,32 @@ func (o *namespaceTransformer) Transform(m resmap.ResMap) error {
 		}
 
 	}
+	o.updateClusterRoleBinding(m)
 	return nil
 }
+
+func (o *namespaceTransformer) updateClusterRoleBinding(m resmap.ResMap) {
+	for id := range m {
+		if id.Gvk().Kind == "ClusterRoleBinding" {
+			objMap := m[id].UnstructuredContent()
+			subjects := objMap["subjects"].([]interface{})
+			for i := range subjects {
+				subject := subjects[i].(map[string]interface{})
+				kind, foundk := subject["kind"]
+				name, foundn := subject["name"]
+				if foundk && foundn {
```

Comment:
Remove the `foundn` conditional to support adding namespaces to service accounts that are missing namespace?

Reply:
Name is used to find the referred object. It is not optional. In the following example, `name` is default.
```
subjects:
- kind: ServiceAccount
  name: default
  namespace: system
```

Reply:
Yep, sorry I missed that `foundn` was talking about `name`, not `namespace`.